__test__/containers/UserMenu.test.js

```diff
@@ -8,35 +8,43 @@ import { StyleSheetTestUtils } from "aphrodite";
 import { UserMenu } from "../../src/containers/UserMenu";
 import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
 
+function getData(isSuperAdmin = "false") {
+  return {
+    currentUser: {
+      id: 1,
+      displayName: "TestName",
+      email: "test@test.com",
+      is_superadmin: isSuperAdmin,
```

Comment:
is_superadmin should probably be `isSuperAdmin` here to keep to the camelCase norm for graphql

Reply:
done